tools.yml

```diff
@@ -1920,9 +2098,17 @@ tools:
   toolshed.g2.bx.psu.edu/repos/mbernt/maxbin2/maxbin2/.*:
     cores: 10
     mem: 48
+  toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_RPKM_saturation/.*:
+    cores: 8
   toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_bam2wig/.*:
     cores: 8
     mem: 16
+  toolshed.g2.bx.psu.edu/repos/nilesh/rseqc/rseqc_geneBody_coverage/.*:
```

Comment:
its not using Galaxy slots afaik: https://github.com/galaxyproject/tools-iuc/blob/8a91236cee4d408ae2b53a3e9b6daebc332d631a/tools/rseqc/geneBody_coverage.xml